pvm/ts/src/engine/texasHoldem-ante-headsup.test.ts

```
@@ -147,7 +147,7 @@ describe("Texas Holdem - Ante - Heads Up", () => {
             expect(game.getPlayer("0x1fa53E96ad33C6Eaeebff8D1d83c95Fcd7ba9dac")).toBeDefined();
         });
 
-        it("should do end to end", () => {
+        it.only("should do end to end", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)